java/test/jmri/jmrit/decoderdefn/MissingTypeTest.java

```diff
@@ -0,0 +1,96 @@
+package jmri.jmrit.decoderdefn;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import jmri.jmrit.XmlFile;
+
+import org.jdom2.Attribute;
+import org.jdom2.Element;
+import org.jdom2.JDOMException;
+import org.jdom2.filter.ElementFilter;
+import org.junit.jupiter.api.*;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Checks for missing content in a <variable> element
+ *
+ * @author Bob Jacobsen Copyright 2010
+ * @since 2.9.3
+ */
+public class MissingTypeTest {
+
+    @Test
+    public void testForDuplicateModels() throws JDOMException, IOException {
```

Comment:
No need to change, but the name of the method seems misleading. Copy/paste issue?